tests/ballerina-compiler-api-test/src/test/java/io/ballerina/semantic/api/test/actions/SymbolsInResourceAccessActionTest.java

```diff
@@ -66,22 +69,46 @@ public void testResourceAccessAction(int line, int col, String name, Map<String,
         assertEquals(functionTypeSymbol.typeKind(), TypeDescKind.FUNCTION);
     }
 
-    @DataProvider(name = "SymbolPosInResourceAccessAction")
-    public Object[][] getPathSegmentInfo() {
+    @DataProvider(name = "ResourceAccessActionPosition")
+    public Object[][] getResourceAccessActionPosAndPathSegments() {
         return new Object[][]{
-                {40, 23, "get", Map.of(
+                {40, 34, "get", Map.of(
                         "foo", PathSegment.Kind.NAMED_SEGMENT,
                         "bar", PathSegment.Kind.NAMED_SEGMENT,
                         "[string id]", PathSegment.Kind.PATH_PARAMETER)},
-                {41, 14, "get", Map.of(
+                {41, 25, "get", Map.of(
                         "foo", PathSegment.Kind.NAMED_SEGMENT,
                         "[string id]", PathSegment.Kind.PATH_PARAMETER)},
-                {42, 17, "post", Map.of(
+                {42, 28, "post", Map.of(
                         "bar", PathSegment.Kind.NAMED_SEGMENT,
                         "[string... ids]", PathSegment.Kind.PATH_REST_PARAMETER)},
         };
     }
 
+    @Test(dataProvider = "SymbolPosInResourceAccessAction")
+    public void testSymbolsInResourceAccessAction(int line, int col, String name, SymbolKind symbolKind) {
+        Optional<Symbol> symbol = model.symbol(srcFile, LinePosition.from(line, col));
+        if (name == null) {
+            assertTrue(symbol.isEmpty());
+            return;
+        }
```

Comment:
```suggestion
        }
        
```
Better add a new line.